src/VehicleSelector/VehicleSelector.tsx

```diff
@@ -223,7 +223,10 @@ function VehicleSelector({
             }
           }}
           size={size}
-          value={selectedVariants}
+          // if array is empty set to null to avoid rendereing the clear button
```

Comment:
spelling and punctuation suggestion

```suggestion
          // if array is empty, set value to null to avoid rendering the clear button
```